### PR TITLE
Sidebar(docs): added tabindex to scrollable examples

### LINKED ISSUE #6322: Fix sidebar overflow example keyboard issue and add documentation
**Describe the issue. What is the expected and unexpected behavior?**
Currently the sidebar overflow examples are not keyboard accessible. We should fix the examples by adding a `tabIndex="0"` to any examples that have scrollable containers. It would also make sense to add a comment to the example about adding the `tabIndex` to the scrollable contents when there is overflow.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Tab to the overflow sidebar examples and notice that you can't get to any hidden/scrollable content by keyboard.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #6322

### CODE REVIEW

**Review by jessiehuff — APPROVED** [review]
LGTM! :)

File: packages/react-core/src/components/Sidebar/examples/Sidebar.md [hunk 1]
```diff
@@ -56,11 +56,12 @@ import { Sidebar, SidebarContent, SidebarPanel } from '@patternfly/react-core';
 ```
 
 ### Sticky panel
+A `tabindex` should be added to the `Sidbear` when there is scrollable content in order for the  overflow content to be accessible by keyboard.
```
Comment (evwilkin) [inline packages/react-core/src/components/Sidebar/examples/Sidebar.md:59]: Typo: "Sidebar" spelling

```suggestion
A `tabindex` should be added to the `Sidebar` when there is scrollable content in order for the  overflow content to be accessible by keyboard.
```

File: packages/react-core/src/components/Sidebar/examples/Sidebar.md [hunk 2]
```diff
@@ -73,11 +74,12 @@ import { Sidebar, SidebarContent, SidebarPanel } from '@patternfly/react-core';
 ```
 
 ### Static panel
+A `tabindex` should be added to the `Sidbear` when there is scrollable content in order for the  overflow content to be accessible by keyboard.
```
Comment (evwilkin) [inline packages/react-core/src/components/Sidebar/examples/Sidebar.md:77]: Typo: "Sidebar" spelling

```suggestion
A `tabindex` should be added to the `Sidebar` when there is scrollable content in order for the  overflow content to be accessible by keyboard.
```

File: packages/react-core/src/components/Sidebar/examples/Sidebar.md [hunk 3]
```diff
@@ -90,11 +92,12 @@ import { Sidebar, SidebarContent, SidebarPanel } from '@patternfly/react-core';
 ```
 
 ### Responsive panel width
+A `tabindex` should be added to the `Sidbear` when there is scrollable content in order for the  overflow content to be accessible by keyboard.
```
Comment (evwilkin) [inline packages/react-core/src/components/Sidebar/examples/Sidebar.md:95]: Typo: "Sidebar" spelling

```suggestion
A `tabindex` should be added to the `Sidebar` when there is scrollable content in order for the  overflow content to be accessible by keyboard.
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Typo - `Sidbear` should be `Sidebar` in example captions.

**Review by nicolethoen — APPROVED** [review]
Looks good once the typos are fixed :)

**Review by evwilkin — APPROVED** [review]
LGTM thanks!

**Review by jessiehuff — APPROVED** [review]

**Pull request merged**